python/lsst/ts/watcher/alarm.py

```diff
@@ -36,6 +39,32 @@ class Alarm:
         Name of alarm. This must be unique among all alarms
         and should be of the form system.[subsystem....]_name
         so that groups of related alarms can be acknowledged.
+
+    Attributes
+    ----------
+    name : str
+        Name of alarm.
+    callback : callable, optional
+        A function to call when the alarm state changes.
+        The function receives one argument: this alarm.
+        None, the default, means no callback.
+    auto_acknowledge_delay : float
+        The delay (seconds) after which an alarm will be automatically
+        acknowledged. Never if 0, the default.
+    auto_unacknowledge_delay : float
+        The delay (seconds) after which an alarm will be automatically
+        unacknowleddged. Never if 0, the default.
+    escalate_to : str
+        Who to escalate this alarm to. Do not escalate if blank (the default).
```

Comment:
You are using both `, the default.` and `(the default).`. Maybe choose one and stick to that for consistency?